_rules/printable-characters-shortcut-ffbc54.md

```diff
@@ -0,0 +1,575 @@
+---
+id: ffbc54
+name: No keyboard shortcuts use only printable characters
+rule_type: atomic
+description: |
+  This rule checks that if keyboard shortcuts are implemented using only printable characters, then there is a mechanism to disable the shortcut, or to remap the shortcut to use one or more non-printable characters keys, or the shortcut for a user interface component is only available when that component has focus.
+accessibility_requirements:
+  wcag21:2.1.4: # Character Key Shortcuts (A)
+    forConformance: true
+    failed: not satisfied
+    passed: satisfied
+    inapplicable: further testing needed
+acknowledgements:
+  authors:
+    - João Vicente
+    - Carlos Duarte
+---
+
+## Applicability
+
+The rule applies to any [keyboard event][]: 
+ - where the attribute `key` is a [printable character][] key; and
+ - the attribute `isComposing` is set to `false`; or the [keyboard event][] is part of an [composition event][] whose attribute `data` includes at least one [non-printable character][] key; and
+ - is [dispatched][] to an [event target][] within a [HTML document][].
+
+## Expectation
+
+For each test target:
+ - an [instrument][] to prevent the outcome of the [keyboard event][] is available; or
```

Comment:
"prevent" might need some more explanation.

Also, this is a bit weird. The goal of the instrument is to "permanently" disable the event (or until toggled again). This sentence looks like "quick, I need to click on that button to prevent the key I just pressed from triggering stuff".
I feel this come from a mix between "keyboard event" being both the global thingie (the shortcut, i.e. the fact that the page reacts to me pressing some key), and the individual events that are fired every time one key is pressed.

Now, I do agree that "keyboard event" as defined refers to the global thingie and not to an individual event being fired… 🤔